src/_packagedcode/pypi.py

```diff
@@ -684,10 +696,44 @@ def parse(cls, location):
 
         metadata = {}
         parser = ConfigParser()
+        dependent_packages = []
         with open(location) as f:
             parser.read_file(f)
-
         for section in parser.values():
+            if section.name == 'options':
+                scope_by_sub_section = {
+                    "install_requires": "install",
+                    "tests_require": "test",
+                    "setup_requires": "setup",
+                    "python_requires": "python",
+                }
+                for sub_section in scope_by_sub_section:
```

Comment:
It is IMHO simpler and cleaner to iterate on key + value at once in this case:
```suggestion
                for sub_section, scope in scope_by_sub_section.items():
```